plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraPageSink.java

```diff
@@ -185,6 +192,9 @@ else if (VARBINARY.equals(type)) {
         else if (UuidType.UUID.equals(type)) {
             values.add(trinoUuidToJavaUuid(type.getSlice(block, position)));
         }
+        else if (typeManager.getType(new TypeSignature(StandardTypes.IPADDRESS)).equals(type)) {
```

Comment:
Please consider adding `boolean isIpAddressType(Type type)` to `CassandraTypeManager` and use here.